flexmeasures/utils/unit_utils.py

```diff
@@ -207,6 +208,25 @@ def is_energy_price_unit(unit: str) -> bool:
     return False
 
 
+def convert_time_units(
+    data: Union[tb.BeliefsSeries, pd.Series, List[Union[int, float]], int, float],
+    from_unit: str,
+    to_unit: str,
+):
+    """Convert data with datetime or timedelta dtypes to float values.
+
+    Use Unix epoch or the requested time unit, respectively.
+    """
+    if from_unit == "datetime":
+        return (
+            pd.to_datetime(data, utc=True) - pd.Timestamp("1970-01-01", tz="utc")
+        ) // pd.Timedelta("1s")
+    if from_unit == "timedelta":
+        if to_unit[0].isdigit():
```

Comment:
What does denote in to_unit? An ISO duration?

Reply:
Can you rephrase your question for my understanding?

Reply:
`to_unit[0].isdigit()` tells you something, but what?